### PR TITLE
Fixes #19121: Cleanup apache_acl bundle in the system technique DistributePolicy

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/19121

### CODE REVIEW

File: techniques/system/distributePolicy/1.0/apache-acl.cf [hunk 1]
```diff
@@ -2,14 +2,24 @@ bundle agent apache_acl
 {
 
   vars:
-      "destination"        string => "${g.rudder_var}/lib/ssl/";
+      "technique_name"     string => "DistributePolicy";
+      "trackingkey   "     string => "${distribute_policy_common.directiveId}";
```
Comment (ncharles) [inline techniques/system/distributePolicy/1.0/apache-acl.cf:6]: extra space in the " " here

File: techniques/system/distributePolicy/1.0/apache-acl.cf [hunk 2]
```diff
@@ -2,14 +2,24 @@ bundle agent apache_acl
...
+      "component"          string => "Configure apache ACL";
+
       "ssl_ca_file"        string => "nodescerts.pem";
-      "ssl_ca_size"        string => filestat("${this.promise_dirname}/${ssl_ca_file}", "size");
+
+      "destination_1"      string => "${g.rudder_var}/lib/ssl/${ssl_ca_file}";
```
Comment (ncharles) [inline techniques/system/distributePolicy/1.0/apache-acl.cf:11]: i would love to have better naming that _1 and _2

**Review by ncharles — CHANGES_REQUESTED** [review]
some minor changes

**Comment by Fdall** [comment]
PR updated with a new commit

**Review by ncharles — APPROVED** [review]

**Comment by Fdall** [comment]
OK, squash merging this PR

**Pull request merged**